client/src/components/modal/mosaic_top_modal.jsx

```diff
@@ -0,0 +1,29 @@
+import React from 'react';
+
+import {TopFrameStyle} from '../../styles/Modal.js';
+
+export const TopModal = ({photos}) => {
```

Comment:
Reviewing: client/src/components/modal/mosaic_top_modal.jsx

- this looks cool, so instead of declaring the variable, and then saying "export default variable" you're doing all of it in the same line with export const TopModal? Hope i'm understanding this currently, and if I am, I'm curious, will you be implementing this same format to your other components or only this one? If you already have, I might have missed it.

Reply:
So I'm only doing that if it's a function or have multiple items I want to export on one page and don't want to use all of them as methods. The react elements I'm keeping as default though. 

Also, I'm also starting to do styling objects so that I can add them inLine which from what I hear is always faster. That's what all the exports are in the style/Modal.js are.